### ISSUE TITLE
Cursor theme doesn't do anything

### ISSUE THREAD

**Issue by pastypastypasty** [snwh/paper-icon-theme#660]
Change my cursor theme in Gnome Tweak Tool doesn't do anything. It says that the cursor theme is Paper, but the cursor is just the default Adwaita cursor. Other cursor themes work properly.

**Comment by snwh** [comment]
Are you sure? The main Paper cursor is quite similar to Adwaitas

**Comment by pastypastypasty** [comment]
Yes, I can tell because of the resize window icons. Actually, last night, I
realized the cursor theme works in virtual box windows, but no other
applications. Really weird.

**Comment by snwh** [comment]
This could be something to do with Wayland, I don't know.

**Comment by vol24pl** [comment]
Same issue.

"Terminal" is the only app where the paper cursor is applied, otherwise adwaita is applied.

**Comment by furquan-lp** [comment]
@vol24pl Have you installed Paper icon theme system-wide (i.e. in `/usr/share/icons`)?

**Comment by vol24pl** [comment]
sry for late reply. Busy month! 

Yes, paper is in /usr/share/icons

Gnome version 3.22.2
Base system Debian 9 (stretch) 64-bit
